modules/stanford_paragraph_card/stanford_paragraph_card.module

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function stanford_paragraph_card_preprocess_paragraph(&$variables) {
+  // This function is not running on several paragraph types.
+  // Could be related to: https://www.drupal.org/project/paragraphs/issues/2897906
+  // It is likely due to something in the way that display suite handles
+  // rendering the patterns. To get around this, we have to use
+  // hook_preprocess_ds_entity_view() instead.
+}
+
+/**
+ * Implements hook_preprocess_ds_entity_view().
+ *
+ * Provide the logic for moving the label (link title) from the button_label
+ * region in the card component to the cta_label region.
+ */
+function stanford_paragraph_card_preprocess_ds_entity_view(array &$variables) {
+
+  // You can stop here if this is not a paragraph type.
+  if (!isset($variables['content']['#paragraph'])) {
+    return;
+  }
+
+  // Define the paragraph object variable for shorter code.
+  $paragraph = $variables['content']['#paragraph'];
+
+  // You can also stop if the paragraph is not a card.
+  if ($paragraph->getType() !== "stanford_card") {
```

Comment:
shouldn't this be `->bundle()`

Reply:
https://git.drupalcode.org/project/paragraphs/blob/8.x-1.x/src/ParagraphInterface.php#L102 

_shrugs_

Reply:
That method just calls the core method.
https://git.drupalcode.org/project/paragraphs/blob/8.x-1.x/src/Entity/Paragraph.php#L304

Reply:
I think using `->bundle()` will be more universal and easier to grep if necessary.

Reply:
https://git.drupalcode.org/project/paragraphs/blob/8.x-1.x/src/Entity/Paragraph.php#L303

Not sure why they wrapped that.

Reply:
yeah kinda dumb IMO

Reply:
> I think using ->bundle() will be more universal and easier to grep if necessary.

yeah, I can get on board with that.